browser/widevine/widevine_utils.cc

```diff
@@ -22,17 +25,34 @@
 #include "brave/browser/widevine/brave_widevine_bundle_manager.h"
 #include "chrome/browser/lifetime/application_lifetime.h"
 #include "chrome/browser/ui/browser_finder.h"
-#include "chrome/browser/ui/browser_window.h"
 #include "chrome/browser/ui/tabs/tab_strip_model.h"
 #endif
 
 #if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
 #include "chrome/browser/component_updater/widevine_cdm_component_installer.h"
-#include "chrome/browser/profiles/profile_manager.h"
 #endif
 
-#if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+using content::BrowserThread;
+
 namespace {
+
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) || BUILDFLAG(BUNDLE_WIDEVINE_CDM)
+PrefService* GetLocalState() {
```

Comment:
I don't think this method needs to exist at all and certainly doesn't need to be protected by a buildflag. `g_browser_process->local_state()` is always available

Reply:
Fixed.